packages/core/src/new-api/types/module.ts

```diff
@@ -225,6 +239,22 @@ export interface ReadEventArgumentFuture {
   eventIndex: number;
 }
 
+/**
+ * A future that represents sending arbitrary data to the EVM.
+ *
+ * @beta
+ */
+export interface SendDataFuture {
+  type: FutureType.SEND_DATA;
+  id: string;
+  module: IgnitionModule;
+  dependencies: Set<Future>;
+  to: string | AddressResolvableFuture;
+  data: string;
```

Comment:
Shouldn't this be optional? Or are we going to use `"0x"` as default?

Reply:
Yep, good catch